modules/policy-shared-responsibility.adoc

```diff
@@ -96,8 +96,8 @@ Red Hat is responsible for enabling changes to the cluster infrastructure and se
 |===
 
 [id="identity-access-management_{context}"]
-== Identity and access management
-The Identity and Access Management matrix includes responsibilities for managing authorized access to clusters, applications, and infrastructure resources. This includes tasks such as providing access control mechanisms, authentication, authorization, and managing access to resources.
+== Access and identity authorization
+The Access and Identity Authorization matrix includes responsibilities for managing authorized access to clusters, applications, and infrastructure resources. This includes tasks such as providing access control mechanisms, authentication, authorization, and managing access to resources.
```

Comment:
The capitalization of "Access and Identity Authorization" should be set to lowercase.

Reply:
Hello @eohartman,
    Thanks for the input. I was wondering if the original text (with caps) was incorrect, but decided to make my changes and go with the original formatting. I will make the requested changes asap and await our review. Thanks!